src/workflow/WorkflowApproval.js

```diff
@@ -5,37 +5,74 @@ import Row from 'react-bootstrap/Row'
 import Col from 'react-bootstrap/Col'
 import Snackbar from '@material-ui/core/Snackbar';
 import { useParams } from 'react-router-dom'
+import Table from 'react-bootstrap/Table'
+import { useLocalStorage } from '../common'
 
 export const WorkflowApproval = () => {
   const rowStyle =
   {
     display: 'flex',
-    justifyContent: 'center'
+    justifyContent: 'start'
   }
 
-  const { id, stage } = useParams()
+  // const { id, stage } = useParams()
   const [snackbar, setSnackBar] = useState(false);
   const [snackbarMessage, setSnackbarMessage] = useState('')
+  const [workflows] = useLocalStorage('ws:workflows', [])
+  // const [showhistory, setShowHistory] = useState([])
+  const [showApprovalTab,setShowApprovalTab] = useState(false) // this needs to be deleted before merge
+  const [workflowId, setWorkflowId] = useState('')
 
-  const handleApproval = () => {
+  const historyData = {
```

Comment:
Do you really need this structure for the `history` property? I think it can simply be an array of all the levels. Like this - `['one'. 'two', 'three']`

Reply:
Agree with the thought. Made the necessary changes.